components/omega/OmegaBuild.cmake

```
@@ -34,64 +36,124 @@ macro(common)
     set(OMEGA_LINK_OPTIONS "")
   endif()
 
-  if(NOT Python_FOUND)
-    find_package (Python COMPONENTS Interpreter)
+endmacro()
 
-    if(NOT Python_FOUND)
-      message(FATAL_ERROR "Python is not available, CMake will exit." )
-    endif()
-  endif()
+macro(run_bash_command command outvar)
+
+  execute_process(
+	COMMAND bash -c "${command}"
+	OUTPUT_VARIABLE ${outvar}
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
 
 endmacro()
 
-# Collect machine and compiler info from CIME
-# and detect OMEGA_ARCH and compilers
-macro(init_standalone_build)
+macro(cime_xmlquery query outvar)
 
-  # update CMake configuration with CIME configuration
-  set(_TMP_CMAKE_FILE ${OMEGA_BUILD_DIR}/_Omega.cmake)
-  set(_PY_OPTS "-p;${E3SM_CIME_ROOT};-o;${_TMP_CMAKE_FILE}")
+  run_bash_command("cd ${CASEROOT} && ./xmlquery ${query} --value" ${outvar})
 
-  if(OMEGA_CIME_COMPILER)
-    list(APPEND _PY_OPTS "-c" "${OMEGA_CIME_COMPILER}")
-  endif()
+endmacro()
+
+macro(read_cime_config)
 
-  if(OMEGA_CIME_MACHINE)
-    list(APPEND _PY_OPTS "-m" "${OMEGA_CIME_MACHINE}")
+  set(NEWCASE_COMMAND "${E3SM_ROOT}/cime/scripts/create_newcase \
+    --res T62_oQU120_ais20 \
+    --compset MPAS_LISIO_TEST \
```

Unfortunate timing. We made some fairly big changes to this test in E3SM: https://github.com/E3SM-Project/E3SM/pull/6437

This test is also maybe a little more complicated than we need, since it has active MALI, MPAS-Ocean and MPAS-Seaice.  I would suggest using a C-case test instead, maybe:
```suggestion
    --res T62_oQU120 \
    --compset CMPASO-NYF \
```

This should be safe because it's in the `e3sm_developer` suite:
https://github.com/E3SM-Project/E3SM/blob/ef925d118e756f4f9379423ee4022581dd5aafad/cime_config/tests.py#L302

---

Thanks for the suggestion. It will be included in the next commit.